### ISSUE TITLE
[Abusive mod] Mass banning of me&my friends in the IRC chat, inappropriate behavior.

### ISSUE THREAD

**Issue by gabbie777** [CreeperHost/MineTogether#427]
Though not a bug request, this guy claims to be some kind of site admin in his snarky remarks so I am completely uncomfortable filing a request somewhere he could just take it down before anyone else reviews the report.
If there's somewhere i can submit this complaint that he can't simply take it down, do let me know
(Timestamp ~8:35PM CST, this guy says you keep chat logs of all of it so I encourage you to go look at that instead of the reasons he twists up)
This Draylynn guy was in the IRC chat while I was chilling with some people making memes about discord mods and the topic of CH came up, and trying to keep the conversation going I respectfully noted that I thought CH cut corners a bit as nothing more than a passing remark since I was surprised CH would make a good community feature like an IRC chat and my opinion of them isn't the best(but I obviously wouldn't go on about that in any chat since I don't want to deter players away from your services for my own opinion). He proceeded to pester me 3 times in chat prompting me to continue that line of conversation about 2 minutes after that comment had been said in passing and had been completely replaced in the chat by people and I talking, and I asked why he was so agitated by that one thing. I tried to steer the convo away but for some reason he was incredibly hooked on that and made a passive-aggressive threat that I was "poking the bear" with my chats and that things would happen if I didn't stop. I directly confronted him asking if that was a threat and that I hadn't done anything wrong or even been disrespectful in the slightest, and around 30 seconds later I was banned from the chat(Only what I can assume, he made a snarky remark and I immediately lost all connection to the IRC chat, the window stating that "Unable to chat as you don't have connection" or something along those lines) and so I had my friend on the server at the time get in to the chat to ask what's up since I didn't really know the situation and thought I had just lost connection(since I wasn't really too worried about being banned, I was being pretty respectful and people seemed to be having a good time, he claims that you all have chat logs of it so I would encourage you to go look back on them). He had a good time with people in chat for probably 2-3 minutes before he(memcpy#5294) said the topic of bans came up and that he asked what's up with him banning a friend of his. Before that came up "Draylynn" was being quite condescending with people, stating that people's being there was a "privilege and not a right" and how he could technically ban anyone if he didn't like them right then and there, and got incredibly agitated at my person coming up for some reason stating i was "saying slanderous things against CH with no basis"(I guess saying that I personally think they cut corners in passing is bad or slanderous in some way? cutting corners isnt even always a bad thing). I asked for his discord in call and he relayed that I wanted to talk in discord through the chat, in which he both gave his discord and condescendingly stated that he didn't want to go through discord, and made a snarky remark that "I won't "cut corners" for him, he appeal like all the rest" effectively continuing that one thing that like 2 people saw for what would then be around 15 minutes, and immediately banned my friend when I friend requested him on discord to speak, his only reasoning being "I told him I wouldn't talk on discord", though he freely gave his discord on his own consent? making two false bans by then(or at the least 1, if saying that I thought CH cuts corners is a bannable offence) and he kept falling back on Wheaton's law or something about "Don't be a dick", which if he wouldn't have made an incredibly rude comment on discord, we would've noted that we made sure the entire time to be professional and respectful to him in asking why he banned me since I was still trying to pinpoint why he would've done something like that(i mean hell, I hadn't even made any strikes at him or anything and just kind of took all of the boasting he had about his coderman debugging skills and reciprocated with my experience as a software engineer)

I do hope this can be resolved since at the end the call between all of us ended up being around 9 people(10 now) and a good amount of them are deciding to chargeback their subscriptions for the time being and voice their grievances with him in his DM's, which he has just said is "cyberbullying" and has made no attempt to resolve his incredibly unprofessional behavior as a self-proclaimed employee and representative of yours and downright insulting other people.

![image_2020-12-19_214609](https://user-images.githubusercontent.com/9136885/102704631-9a115000-4243-11eb-92f5-a0534ab41ac2.png)

If you have discord and are reviewing this, i'd be happy to hop in a DM and help work this through with you! My discord is Electr099#5713. I'll link his reaction on discord here. I hold a level of respect for CH since it is a big server host and hope that this is a one-off occasion with the employee and that this isn't a norm

(Again, not even asking for an unban, just a review of this mods action since he seems to be quite off the handle and I fear other players with less emotional security than me might've been affected due to his flaunting about being able to ban anyone in chat)

Highlighting this part too, I don't think adding on discord involves being a dick or breaks rules and all of us but 2 people have been mostly respectful to him through all of this.

![image_2020-12-19_215717](https://user-images.githubusercontent.com/9136885/102704771-283a0600-4245-11eb-98c6-f4ad0d246b6f.png)

My friends messages with him, which he had no backup but to block them on..._Because innocent people obviously go around blocking everyone right?_ I'm surprised he's taken this hostile of a stance too, I added him on discord because I thought he was at-least capable of discussing it, instead being met with passive-aggressive comments like "How hard is it to follow instructions" and my friends being banned for trying to help me out.

![image_2020-12-19_215332](https://user-images.githubusercontent.com/9136885/102704716-a21dbf80-4244-11eb-9127-cbdcd5ca697e.png)

(some less respectful ones since he seemed adamant on just banning&blocking all of us to make the problem disappear instead of talking about it with anyone, leading to them deciding to chargeback their servers)

![image_2020-12-19_220056](https://user-images.githubusercontent.com/9136885/102704806-aac2c580-4245-11eb-818c-3c3c44b39516.png)

TL;DR: Your company representative falsebanned my friends and I, and made a bunch of nasty comments to us, please review his actions as a moderator as even other people in the chat were commenting on how his actions are representing the company.

**Comment by Cloudhunter** [comment]
I've had a cursory look into this, and from what has been said by Draylynn - whom is a she, not he, btw - things done by you and your friends were much more abusive than anything you have stated here. It's also patently obvious that you hounded her - and got multiple messages sent to her from different friends accounts - despite clear statements that the ban wasn't to be discussed on discord, and that it could be appealed at minetogether.io

I'd also like to remind your friends that a chargeback when you have received a service is fraudulent and all chargebacks will be fought with evidence and won.

I will be closing and locking this as this isn't a good place for discussion.

**Comment by ThePaul-T** [comment]
Good morning,

If you and/or your friends would like to provide your server names here, I would personally like to extend an appropriate response as specified via our terms of service.

However, I will also note, given the actions of your "friends" on this GitHub and otherwise, neither you nor they now have any chance of being unbanned. If things had been left with just an issue requesting someone looked into what happened, an actual investigation would've taken place, I would've read the chat logs back and dealt with this as was required in the morning.

Regards,
Paul
